test/aura/AuraAvatarMultiTokenTest.t.sol

```diff
@@ -701,22 +751,105 @@ contract AuraAvatarTwoTokenTest is Test, AuraAvatarUtils {
     }
 
     function test_claimRewardsAndSendToOwner_noRewards() public {
-        vm.expectRevert(AuraAvatarTwoToken.NoRewards.selector);
+        vm.expectRevert(AuraAvatarMultiToken.NoRewards.selector);
         vm.prank(owner);
         avatar.claimRewardsAndSendToOwner();
     }
 
+    function test_addBptPositionInfo__permissions() public {
+        vm.expectRevert("Ownable: caller is not the owner");
+        avatar.addBptPositionInfo(21);
+    }
+
+    function test_addBptPositionInfo() public {
```

Comment:
I would use this chance to test that the other sets also increased in size accordingly and a step further would be to checck that the new asset and rewardPool stored match those from the Booster for this PID

Reply:
added suggestions in the test in 20af3b87a884a027b7edeefff641fb286e1c5a05